index.js

```diff
@@ -48,7 +48,7 @@ async function buildArchPackage(os, cpu, version, pre) {
   debug("building architecture specific package", os, cpu, version, pre)
 
   const platform = os == 'win' ? 'win32' : os
-  const arch = os == 'win' && cpu == 'ia32' ? 'x86' : cpu
+  const arch = os == 'win' && cpu == 'ia32' ? ['x86', 'ia32'] : cpu
```

Comment:
Where are these coming from / used? Is there a nodejs binary release marked ia32?

Reply:
This change is necessary for 32bit Windows to install the npm package `node`, otherwise `npm` will throw
```
error C:\Users\IEUser\babel\packages\babel-preset-env\node_modules\node: Command failed.
Exit code: 1
Command: node installArchSpecificPackage
Arguments: 
Directory: C:\Users\IEUser\babel\packages\babel-preset-env\node_modules\node
Output:
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for node-win-x86@12.13.1: wanted {"os":"win32","arch":"x86"} (current: {"os":"win3
2","arch":"ia32"})
npm ERR! notsup Valid OS:    win32
npm ERR! notsup Valid Arch:  x86
npm ERR! notsup Actual OS:   win32
npm ERR! notsup Actual Arch: ia32
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\IEUser\AppData\Roaming\npm-cache\_logs\2019-12-17T01_34_23_329Z-debug.log
internal/modules/cjs/loader.js:628
    throw err;
    ^
Error: Cannot find module 'node-win-x86/package.json'
```

As long as `ia32` is mapped to install `node-win-x86` instead, we should state this at its package.json that `cpus: ["x86", "ia32"]` is supported.